### PR TITLE
Added "Document Query System with LlamaIndex and LLM Integration" [issue #836]

### LINKED ISSUE #836: I want to add "Document Query System using LlamaIndex"
[](url)Project Description:

This project focuses on building a document query system that allows users to extract relevant information from a large PDF document through natural language queries. It uses LlamaIndex (formerly GPT Index) for parsing and indexing the data and integrates Hugging Face's HuggingFaceInferenceAPI for both language modeling and embeddings.
Screenshot (115)

I would like to work on this . Please assign to me

### PR DESCRIPTION
**Pull request by madhavi-peddireddy**
Description
This pull request addresses 

Fixes #836  

Changes Made
Created New folder named " Generative AI"
Under the folder added the notebook

### CODE REVIEW

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Please ensure that your code is located in the `docs` folder. Please carefully read our CONTRIBUTE.md file for guidelines and resources. We only accept .md files for our documentation.

**Comment by madhavi-peddireddy** [comment]
I have made the necessary changes. The code is now located in the docs folder, and I've ensured compliance with the guidelines and resources outlined in the CONTRIBUTE.md file. All documentation has been provided in .md format as required.

File: docs/Document_Query using LlamaIndex/README (1).md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# 📖 Document Query System with LlamaIndex and LLM Integration 🚀
```
Comment (ajay-dhangar) [inline docs/Document_Query using LlamaIndex/README (1).md:12]: We need on top

```md
---
id: ___
title: ___
sidebar_label: ___
sidebar_position: ___
description: ___
tags: [___, __, ]
---
```

### Detailed Guidelines

- id: Unique identifier for the route.
- title: Title of the problem.
- sidebar_label: Problem number or label for the sidebar.
- tags: Relevant tags for the problem.
- description: A short SEO-friendly description. No HTML tags.

File: docs/Document_Query using LlamaIndex/README (1).md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- 🤖 **LLM Responses**: Utilize LLMs to generate responses based on the retrieved context.
+- 📂 **Persistent Directory**: Save embeddings and indexes for reusability, avoiding redundant computations.
+
+---
+## 🔧 **Installation** 
+
+To get started, clone this repository and install the required packages:
+
+```bash
+!pip -q install llama-index langchain llama-index-readers-web llama-index-embeddings-langchain llama-index-llms-huggingface langchain-community
```
Comment (ajay-dhangar) [inline docs/Document_Query using LlamaIndex/README (1).md:34]: close with ```` ``` ````

File: docs/Document_Query using LlamaIndex/code.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
+## Llama Index
```
Comment (ajay-dhangar) [inline docs/Document_Query using LlamaIndex/code.md:10]: We need on top

```md
---
id: ___
title: ___
sidebar_label: ___
sidebar_position: ___
description: ___
tags: [___, __, ]
---
```

### Detailed Guidelines

- id: Unique identifier for the route.
- title: Title of the problem.
- sidebar_label: Problem number or label for the sidebar.
- tags: Relevant tags for the problem.
- description: A short SEO-friendly description. No HTML tags.

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

File: docs/Document_Query using LlamaIndex/README (1).md (hunk 1 above)
Comment (madhavi-peddireddy) [reply docs/Document_Query using LlamaIndex/README (1).md:12]: Changes made as per your guidance

File: docs/Document_Query using LlamaIndex/code.md (hunk 3 above)
Comment (madhavi-peddireddy) [reply docs/Document_Query using LlamaIndex/code.md:10]: updated

File: docs/Document_Query using LlamaIndex/README (1).md (hunk 2 above)
Comment (madhavi-peddireddy) [reply docs/Document_Query using LlamaIndex/README (1).md:34]: Closed

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**